yarn.lock

```diff
@@ -3465,7 +3465,19 @@ __metadata:
   languageName: unknown
   linkType: soft
 
-"@metamask/profile-sync-controller@^0.0.0, @metamask/profile-sync-controller@workspace:packages/profile-sync-controller":
+"@metamask/profile-sync-controller@npm:^0.0.0":
```

Comment:
The `@metamask/notification-services-controller` still relies on the v0.0.0 of profile sync. We will make a separate release for `@metamask/notification-services-controller`